source/com.android.billingclient/billing/Transforms/Metadata.xml

```diff
@@ -2,23 +2,10 @@
 	<!-- namespaces -->
 	<attr path="/api/package[@name='com.android.billingclient']" name="managedName">Android.BillingClient</attr>
 	<attr path="/api/package[@name='com.android.billingclient.api']" name="managedName">Android.BillingClient.Api</attr>
-	<attr path="/api/package[@name='com.android.billingclient.util']" name="managedName">Android.BillingClient.Util</attr>
```

Comment:
The com.android.billingclient.util package doesnt exist any more - only com.android.billingclient.api: https://developer.android.com/reference/com/android/billingclient/packages.

This warning shows: warning BG8A04: Metadata.xml element '<attr path="/api/package[@name='com.android.billingclient.util']" />' matched no nodes.

Reply:
Same with com.android.vending.billing in the next line

Reply:
> This warning shows: warning BG8A04: Metadata.xml element '' matched no nodes.

This is usually OK. I mean it is not - performance-wise and sustainability (processor ticks => energy => CO2), but if the package is not there it will not be surfaced. Matadata does not hit it.

Reply:
got it